library/src/main/java/com/google/android/exoplayer/text/webvtt/WebvttParser.java

```diff
@@ -46,6 +47,10 @@
   private static final Pattern WEBVTT_FILE_HEADER =
       Pattern.compile(WEBVTT_FILE_HEADER_STRING);
 
+  private static final String WEBVTT_COMMENT_BLOCK_STRING = "^NOTE((\\u0020|\u0009).*)?$";
```

Comment:
You are right, I think it was a typo + copy&paste error in my side. Both should have just one single backslash. I am applying the fix in my branch and will add it to the PR.